<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewdatabasewindow</class>
 <widget class="QMainWindow" name="viewdatabasewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #32383D;
                                  color: white;
                                  }
                                  QPushButton:pressed {
                                  background-color: rgba(255, 255, 255, 0.2);</string>
    </property>
    <property name="text">
     <string>View Sensor Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    QPushButton {
        background-color: white;
        color: #007ACC; /* Text color */
        border-radius: 25px; /* Rounded corners */
        font-weight: bold;
        font-size: 16px;
    }
    QPushButton:hover {
        background-color: #E0E0E0; /* Hover effect */
        box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.2);
    }
    QPushButton:pressed {
        background-color: #B0B0B0; /* Click effect */
    }</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>511</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>190</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #32383D;
                                  color: white;
                                  }
                                  QPushButton:pressed {
                                  background-color: rgba(255, 255, 255, 0.2);</string>
    </property>
    <property name="text">
     <string>View Sensor Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>320</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #32383D;
                                  color: white;
                                  }
                                  QPushButton:pressed {
                                  background-color: rgba(255, 255, 255, 0.2);</string>
    </property>
    <property name="text">
     <string>View Sensor Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #32383D;
                                  color: white;
                                  }
                                  QPushButton:pressed {
                                  background-color: rgba(255, 255, 255, 0.2);</string>
    </property>
    <property name="text">
     <string>View Sensor Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #32383D;
                                  color: white;
                                  }
                                  QPushButton:pressed {
                                  background-color: rgba(255, 255, 255, 0.2);</string>
    </property>
    <property name="text">
     <string>View Sensor Database</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
